src/Terminal/Parser.ts

```diff
@@ -1,12 +1,21 @@
 import { KEY } from "../utils/helpers/keyCodes";
 import { substituteAliases } from "../Alias";
-// Helper function that checks if an argument (which is a string) is a valid number
-function isNumber(str: string): boolean {
-  if (typeof str != "string") {
-    return false;
-  } // Only process strings
-  return !isNaN(str as unknown as number) && !isNaN(parseFloat(str));
+// Helper function to parse individual arguments into number/boolean/string as appropriate
+function parseArg(arg: string): string | number | boolean {
+  // Handles all numbers including hexadecimal, octal, and binary representations, returning NaN on an unparseable string
+  const asNumber = Number(arg);
+  if (!isNaN(asNumber)) {
```

Comment:
Prefer isFinite unless you want to allow Infinity.

Reply:
I don't see any reason to explicitly forbid Infinity as a numeric argument when all other valid JavaScript numeric literals are now allowed.